packages/cli/openapi-parser/src/schema/convertSchemas.ts

```diff
@@ -169,7 +182,7 @@ export function convertSchemaObject(
                     maxLength: schema.maxLength,
                     pattern: schema.pattern,
                     format: schema.format,
-                    example: getExamplesString(schema)
+                    example: getExamplesString(schema, context.logger, exampleUnderride as string)
```

Comment:
would be valuable to understand why we are still casting here and not checking the type re  `exampleUnderride as string`